### ISSUE TITLE
getbufvar signature incompatible with vim < v7.4

### ISSUE THREAD

**Issue by amirnissim** [airblade/vim-gitgutter#337]
```
$ vim --version
VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Oct 28 2015 19:46:19)
Compiled by root@apple.com
```

```
Error detected while processing function gitgutter#process_buffer..gitgutter#handle_diff..gitgutter#diff#process_hunks..gitgutter#diff#process_hunk..gitgutter#hunk#increment_lines_
added..gitgutter#hunk#summary:
line    1:
E118: Too many arguments for function: getbufvar
E15: Invalid expression: getbufvar(a:bufnr, 'gitgutter_summary', [0,0,0])
```

cf. https://github.com/kopischke/vim-stay/issues/19

**Comment by airblade** [comment]
Since you're on OS X, you could upgrade your Vim via MacVim or Homebrew...

**Comment by groggemans** [comment]
Probably related (OSX, vim <7.4): 
```
Error detected while processing function gitgutter#all..gitgutter#process_buffer:
line   17:
E216: No such group or event: <nomodeline> User GitGutter
```

**Comment by airblade** [comment]
@groggemans Is your vim-gitgutter up to date?  This was fixed in 264d7f3.

**Comment by groggemans** [comment]
Your right, thanks! Thought I updated yesterday, but seems to be a few days ago.

I do think it would be nice if OSX can be supported "out of the box". I have around  30 plugins and vim-gitgutter is one of the 2 that isn't support by the default vim version.

**Comment by airblade** [comment]
Addressed in f15a67f.
